src/scripts/profil.js

```diff
@@ -648,7 +648,7 @@ export default class Profil {
     }
 
     hasSymptomesActuelsReconnus() {
-        return this.symptomes_actuels && !this.symptomes_actuels_autre
+        return !!(this.symptomes_actuels && !this.symptomes_actuels_autre)
```

Comment:
```suggestion
        return Boolean(this.symptomes_actuels && !this.symptomes_actuels_autre)
```

Je trouve cette forme plus explicite et à il me semble que ça fait la même chose (attention à ne pas utiliser `new` !)

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Boolean

Reply:
J’ai hésité entre les deux. Mon penchant personnel va aussi vers la forme plus explicite, mais j’ai eu l’impression que `!!` était plus idiomatique en JS. OK pour changer !